res/graphics/authors.txt

```diff
@@ -253,6 +253,7 @@ italian police type/
     polizia.svg                     modified from https://commons.wikimedia.org/wiki/File:Italian_traffic_signs_-_icona_polizia.svg, Gigillo83, License: CC-BY-SA 3.0
 
 quest/
+    access_point.svg
```

Comment:
Shouldn't this be called something more generic like emergency, or descriptive like plus/cross, given the same thing would also be appropriate for first aid or other emergency quests.

More generally what about adding a map pin to the centre to make it a bit more specific?

Reply:
No, icon is fine

Reply:
Fair enough, is it worth renaming it though to be more descriptive like most of the others are?